src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,37 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health) {
+    super(name, health);
```

Comment:
you don't need health param here, you don't even use it
```suggestion
  constructor(name) {
    super(name);
```